todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.kt

```diff
@@ -40,8 +40,8 @@ class TaskDetailFragment : Fragment() {
     }
 
     private fun setupFab() {
-        activity?.let { act ->
-            act.findViewById<View>(R.id.fab_edit_task).setOnClickListener {
+        activity?.run {
```

Comment:
Sorry, in this case, you don't need `run` or anything at all. Just say activity?.findViewById...